### ISSUE TITLE
Quick Start Guide seems out of date

### ISSUE THREAD

**Issue by evilC** [BeyondTrust/pbis-open#1]
Trying to install on RHEL7...

From https://github.com/BeyondTrust/pbis-open/wiki/docs/PBIS_Open_Quick_Start.pdf

```
Step 1: Download PBIS Open
To download PBIS Open:
1. Browse to www.beyondtrust.com and click Downloads and Free Tools.
```
There appears to be no such link

I tried installing from the repo ( https://repo.pbis.beyondtrust.com/yum.html )  
I got the package installed, but I see no GUI.

**Comment by rbest-bt** [comment]
We are currently working on updating the Quick start guide for the transition to github.

The downloads can be found on the release page:
https://github.com/BeyondTrust/pbis-open/releases

As for a GUI there is only one for the domainjoin process which needs some updating. This can also been done via the command-line:
/opt/pbis/bin/domainjoin-cli

**Comment by evilC** [comment]
Thanks for the prompt reply.  
So does this mean that the YUM repos that are linked from this wiki are not currently distributing the correct version?  
If so, advise removing the links in the wiki to both the QuickStart guide and the repos, to avoid confusion.

**Comment by rbest-bt** [comment]
Release page and repositories are both updated with the same version at release. Both are currently 8.5.1.206

I'll leave this open as a reminder to upload the new quick start guide once it's completed.

**Comment by rbest-bt** [comment]
Quick start guide updated
https://github.com/BeyondTrust/pbis-open/wiki/docs/PBIS_Open_Quick_Start.pdf

**Comment by evilC** [comment]
From the guide:
```
"After the wizard finishes installing PBIS Open, the user interface for joining a domain appears."
```
Did not appear for me.

Printed to the console after running the installer:
```
As root, run domainjoin-gui or domainjoin-cli to join a domain...
[...]
domainjoin-gui: command not found...
```
Rebooted, and still no domainjoin-gui command available.
domainjoin-cli is, however.

**Comment by rbest-bt** [comment]
pbis and domainjoin-cli are available commands in the path but domainjoin-gui is not. I'll bring this up to get removed from the post install text to prevent confusion.

As for the the gui not launching. This would be caused by a missing library in the latest distributions
[rbest-bt@rhel7.r ~]$ /opt/pbis/bin/domainjoin-gui
/opt/pbis/bin/domainjoin-gui: error while loading shared libraries: libpangox-1.0.so.0: cannot open shared object file: No such file or directory

We are aware of this and are planning to update them.

For now I recommend the domainjoin-cli as it's much quicker then the gui anyway:
domainjoin-cli join MYDOMAIN.COM MyJoinAccount

**Comment by evilC** [comment]
After trying to get this to work on RHEL (Tried 7.1, 7.3 beta and 7.3), I gave up after days of trying, but it seems that this is largely to do with RHEL's networking being fundamentally broken.
So I tried Ubuntu.

I ran the pbis-open-8.5.1.206.linux.x86.deb.sh on Ubuntu 16.10 and it broke apt
I can now no longer install anything on Ubuntu, as it always moans that pbis-open "subprocess installed post-installation script returned error exit status 1. Errors were encountered while processing: pbis-open:i386"

I tried searching around on the internet and all the suggestions I can find to fix it don't work.
`apt-get clean` etc, all do nothing to fix this.
